WalletWasabi.Fluent/Helpers/ApplicationUtils.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reactive.Linq;
+using System.Reactive.Threading.Tasks;
+using Avalonia;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public static class ApplicationUtils
+{
+	public static IObservable<string> GetClipboard()
```

Comment:
It's not getting once the contents of clipboard but returns an observable, so better name if like ObserveTextClipboard or something like that

Reply:
I agree with that. Fixed.